CONFIG.md

```diff
@@ -8,26 +8,25 @@ The following environment variables are detected by the solarwinds_apm gem and a
 
 Name | Description | Default
 ---- | ----------- | -------
-`SOLARWINDS_SERVICE_KEY` | API token + service name combination, mandatory for metrics and traces to show in my.appoptics.com |
-`IGNORE_APPOPTICS_WARNING` | tells the solarwinds_apm gem to __not__ output the _missing AppOpticsAPM libraries_ message on stack initialization | `false`
-`APPOPTICS_GEM_VERBOSE` | sets the verbose flag (`AppOpticsAPM::Config[:verbose]`) early in the gem loading process which may output valuable information | `false`
-`APPOPTICS_CUUID` | Allows specifying the customer ID via environment variable to override/bypass the value in `/etc/tracelytics.conf` | `nil`
+`SW_APM_SERVICE_KEY` | API token + service name combination, mandatory for metrics and traces to show in the dashboard |
+`SW_APM_GEM_VERBOSE` | sets the verbose flag (`SolarWindsAPM::Config[:verbose]`) early in the gem loading process which may output valuable information | `false`
+`SW_APM_NO_LIBRARIES_WARNING` | tells the solarwinds_apm gem to __not__ output the _missing SolarWindsAPM libraries_ message on stack initialization | `false`
 
 # Related to Tests
 
 Name | Description | Default
 ---- | ----------- | -------
-`APPOPTICS_GEM_TEST` | puts the gem in test mode to avoid restarting certain background services used in testing.   `false`
+`SW_APM_GEM_TEST` | puts the gem in test mode to avoid restarting certain background services used in testing.   `false`
 `DBTYPE` | For tests on Ruby on Rails, specifies the database type to test against.  `postgres`, `mysql` and `mysql2` are valid options. | `postgres`
-`APPOPTICS_CASSANDRA_SERVER` | specifies the Cassandra server to test against. | `127.0.0.1:9160`
-`APPOPTICS_MONGO_SERVER` | specifies the Mongo server to test against. | `127.0.0.1:27017`
-`APPOPTICS_RABBITMQ_SERVER` | specifies the RabbitMQ server to test against. | `127.0.0.1`
-`APPOPTICS_RABBITMQ_PORT` | port for the RabbitMQ connection. | `5672`
-`APPOPTICS_RABBITMQ_USERNAME` | username for the RabbitMQ connection | `guest`
-`APPOPTICS_RABBITMQ_PASSWORD` | password for the RabbitMQ connection | `guest`
+`SW_APM_CASSANDRA_SERVER` | specifies the Cassandra server to test against. | `127.0.0.1:9160`
```

Comment:
I know this is soon to be moot since we're going to remove ruby support for cassandra, but why the `SW_APM_` prefix here and not like below other server en vars?

Reply:
I just didn't bother, because it will be gone soon in the next PR

Reply:
removed it